src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/InputRightCurlyAloneOrSingleline.java

```diff
@@ -105,7 +105,7 @@ void doDoubleBraceInitialization() {
             put("first", "second");
             put("polygene", "lubricants");
             put("alpha", "betical");
-        }}; // it's ok
+        }}; //violation
```

Comment:
@romani 
If this becomes a violation, then what option should we use to avoid violations for double brace map initializes? I guess there will be no way to do this. Such initializes are evil as for me, however legacy code contains these language structures.